src/client/Microsoft.Identity.Client/Platforms/Features/DesktopOS/StaTaskScheduler.cs

```diff
@@ -13,6 +13,9 @@ namespace Microsoft.Identity.Client.Platforms.Features.DesktopOs
     // This IDisposable class doe not need to implement Dispose method in standard way, because it is sealed.
     // If it ever needs to become inheritable, it should follow the standard pattern as desribed in http://msdn.microsoft.com/en-us/library/fs2xkftw(v=vs.110).aspx.
     /// <summary>Provides a scheduler that uses STA threads.</summary>
+#if NET5_WIN
+    [System.Runtime.Versioning.SupportedOSPlatform("windows")]
```

Comment:
Don't know why the analyzer complains as all these types are internal... Logged a bug on .net team ... https://github.com/dotnet/roslyn-analyzers/issues/4933 but for now, annotating the warnings.